gammapy/irf/psf/parametric.py

```diff
@@ -328,6 +328,7 @@ class EnergyDependentMultiGaussPSF(ParametricPSF):
     tag = "psf_3gauss"
     required_axes = ["energy_true", "offset"]
     required_parameters = ["sigma_1", "sigma_2", "sigma_3", "scale", "ampl_2", "ampl_3"]
+    required_arguments = ["rad", "energy_true", "offset"]
```

Comment:
One should remember that this is gadf specific. Should we keep this simple scheme or start to anticipate a more generic approach?